proto/ai/traceable/agent/config/v1/config.proto

```diff
@@ -16,6 +16,7 @@ message AgentConfig {
   // javaagent has the configs specific to javaagent
   Javaagent javaagent = 7;
   LogConfig logging = 8;
+  MetricsConfig metrics_config = 9;
```

Comment:
Which metrics will be covered in this metrics_config?

Reply:
For now, we just have sampling metrics